### PR TITLE
Only forward audio packets of the loudest speakers in a conference.

### PR DESCRIPTION
**Pull request by jqdrqgnq**

### CODE REVIEW

**Comment by jbg** [comment]
Is there a way to turn this functionality off completely? (e.g. `numLoudest = -1`)

e.g. some deployments are using JVB with custom frontends and client-side mixing based on some external factors, and rely on all audio being forwarded by JVB all the time.

**Comment by bgrozev** [comment]
Jenkins, add to whitelist

File: jvb/src/main/resources/reference.conf [hunk 1]
```diff
@@ -289,6 +289,21 @@ videobridge {
     }
   }
 
+  loudest {
+      # The number of current loudest speakers to route audio packets for.
+      numLoudest = 3
```
Comment (bgrozev) [inline jvb/src/main/resources/reference.conf:294]: Can we have a way to disable the new behavior and default to it?

Comment (jqdrqgnq) [reply jvb/src/main/resources/reference.conf:298]: off by default now.

**Review by bgrozev — APPROVED** [review]

**Pull request merged**